### ISSUE TITLE
Sensor stopped "No session in progress"

### ISSUE THREAD

**Issue by Illiteratus** [NightscoutFoundation/xDrip#1104]
My sensor stopped all of a sudden last night without running into over-time. After inserting a new one, phone got readings but won't display anything. Same thing with the watch, even as SSW3 set as collector. System status says "got data" from watch but no readings shown. When warmup time elapsed just got message on main screen "collecting data" (need at least two readings) and that kept going for hours with no change. 
Changed sensor, nothing. Changed transmitter, thinking it might have died, even though the system said it's receiving data. Still no change....
Many restarts, no use....
Sometime had messages in the log "Raw values not changing-blocking further processing" and hundreds of Task deadlocks on send data and some Nightscout treatments notifications, Task deadlock messages, "noBG? COULDN'T FIND SLOPE".... Etc.

Finally, with no special reason at all, after another sensor restart - and still during the warmup !! - I got the option to do initial calibrations. Not as a message on the main screen, it just appeared on the menu tab !?
As I inserted the BG-data, the readings started to appear on phone and watch while still warming up.....
Right now the warm-up time ended and data is shown correctly, but xdrip still says "sensor stopped". I can only guess that this is due to the activated "Fallback to xDrip" option. Log says all the time NIGHTSCOUT TREATMENTS "Cannot use bloodtest which is not type Finger:Manual" and OB1 says "No session in progress".
Made another finger BG and put it into calibrations. But system says "Calibration rejected: Sensor stopped".
Also updated to the last nightly (Friday). Everything with no use.....

There's something seriously messed up here.
I'm thinking of deleting the whole app and reinstall without importing any settings and do the setup from scratch.....  Maybe this could help as a last solution that comes to my mind.
Should you maybe find another solution, pls share....

**Comment by apeterson22** [comment]
Here's how I got my son's to work:
1) Stop Sensor
2) Start Sensor using code 0000 for 15-20 mins
3) Forget Device
4) Restart phone
5) Stop sensor
6) start sensor with correct old 4 digit code
After 1.7 hr warm up data if flowing. When I hit "Forget Device" I did NOT delete anything from Settings. The transmitter info stayed, it Auth'd after about 10-20 mins. 

WARNING: I was working on getting it to connect (Dexcom hasn't sent new sensors so we have to hope this one last an extra 5 days) for a couple hours, I got this method from reading https://seemycgm.com/2019/08/01/restarting-dexcom-g6-sensors-updated/ and adopting it to being without an iPhone or official Dexcom device, so That is the order I recall doing after 2 failed sensor session after the original sensor session stop Auto-magically from the transmitter. Hope this helps/works.
